cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -13,18 +13,13 @@
 update_centers:
   - id: default
     url: "https://updates.jenkins-ci.org/{{ jenkins_version }}/update-center.json"
-# If set to true, Jenkins will be run over HTTPS (Jenkins on port 8443 with a
-# firewall forward from port 443). If you set this value to true, you MUST
-# override jenkins_ssl_cert and jenkins_ssl_key in a production system or you
-# will not see any improvements in security
-https_enabled: false
```

Comment:
Because, honestly, I don't really see the point of having https disabled if we can automatically generate a self-signed SSL cert when none provided and it would require having 2 separate nginx configuration templates. However if it's really needed I can add it. Let's discuss it tomorrow.